planemo/workflow_lint.py

```diff
@@ -138,9 +145,22 @@ def lint_workflow_artifacts_on_paths(
 def _lint_workflow_artifacts_on_path(
     lint_context: WorkflowLintContext, path: str, lint_args: Dict[str, Union[str, List[str]]]
 ) -> None:
+    iwc_grade = lint_args["iwc_grade"] == "True"
+    if iwc_grade:
```

Comment:
```suggestion
    if lint_args["iwc_grade"]:
```

or alternatively if you like to keep the extra variable:

```
    iwc_grade = lint_args["iwc_grade"]
    if iwc_grade:
```

Reply:
Yes this is what I had before...